### DISCUSSION TITLE
Colored dust particles?

### DISCUSSION THREAD

**Discussion by hugovdev** [Minestom/Minestom#1560] [Help] [answered]
I'm doing this:
`            player.sendPacketToViewersAndSelf(ParticleCreator.createParticlePacket(Particle.DUST, false, currentPos.x(), currentPos.y(), currentPos.z(),
                            0.5f, 0.5f, 0.5f, 0.0F, 15, (binaryWriter) -> {
                                binaryWriter.writeFloat(0.47450980302f);
                                binaryWriter.writeFloat(0.196078431f);
                                binaryWriter.writeFloat(0.78823529262f);
                            }));`

But player gets kicked when getting sent the packet

**Comment by TheMode** [comment, marked as answer]
https://wiki.vg/Protocol#Particle

Apparently require 4 floats.

**Reply by hugovdev** [reply]
Oh shoot! I was literally staring at that page lol hahahah thanks

**Reply by emortaldev** [reply]
Last one is size
